git-multimail/git_multimail.py

```diff
@@ -777,6 +777,9 @@ def get_summary(self):
     def __eq__(self, other):
         return isinstance(other, GitObject) and self.sha1 == other.sha1
 
+    def __ne__(self, other):
+        return isinstance(other, GitObject) and self.sha1 != other.sha1
```

Comment:
This is not the inverse of `__eq__`, which would be

    return not isinstance(other, GitObject) or self.sha1 != other.sha1

or

    return not (self == other)

Note to self: This is actually only needed in Python2. In Python3, if there no `__ne__` is defined, then `__eq__` is called and its result is inverted.

Reply:
I was wondering about [`functools.total_ordering`](https://docs.python.org/3/library/functools.html#functools.total_ordering). I think the latter makes sense, and we can drop it one day if we ever drop Python 2 support, which this library probably won't.